# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/dietlibc/pkg_patch/pkg_wget.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/progress.c~	Fri May 24 14:58:17 2002
+++ ./src/progress.c	Sat Mar 15 20:48:35 2003
@@ -690,7 +690,7 @@
   else if (bp->total_length > 0)
     APPEND_LITERAL ("    ETA --:--");
 
-  assert (p - bp->buffer <= bp->width);
+  assert ((p - bp->buffer) <= bp->width);
 
   while (p < bp->buffer + bp->width)
     *p++ = ' ';
--- ./src/url.c~	Sat Mar 15 19:50:51 2003
+++ ./src/url.c	Sat Mar 15 20:49:43 2003
@@ -205,7 +205,7 @@
 	*p2++ = *p1++;
     }
   *p2 = '\0';
-  assert (p2 - newstr == newlen);
+  assert ((p2 - newstr) == newlen);
 
   return newstr;
 }
@@ -408,7 +408,7 @@
 	}
     }
   *p2 = '\0';
-  assert (p2 - newstr == newlen);
+  assert ((p2 - newstr) == newlen);
   return newstr;
 }
 
@@ -1846,7 +1846,7 @@
   p += fplen;
   *p++ = '\0';
 
-  assert (p - result == size);
+  assert ((p - result) == size);
 
   if (quoted_user && quoted_user != url->user)
     xfree (quoted_user);
@@ -2337,7 +2337,7 @@
 	      *to++ = 'F';
 	    }
 	}
-      assert (to - newname == fsqlen);
+      assert ((to - newname) == fsqlen);
       *to = '\0';
 
       file_sans_qmark = newname;
